lib/tasks/errors.rake

```diff
@@ -0,0 +1,43 @@
+namespace :errors do
+  desc <<-END_DESC
+Fetches log for given request id
+
+This task is useful to get log for given request id from logs in log directory. The request_id is required.
+You can also specify log file where request logs has been saved. Default option is 'production.log'.
+
+Available attributes:
+  * request_id = Request id from 500 page
+  * log_file = specify another log file to lookup (default is 'production.log')
+
+Examples:
+  # foreman-rake errors:fetch_log request_id=aabbccss5 [log_file=development.log]
+  END_DESC
+
+  task :fetch_log => :environment do
```

Comment:
Would this still work if we use different logging backend instead of file (e.g. journald/syslog)?

Reply:
There's easier way to get this from journald/syslog as the request id is one of the field you can filter on. We should probably tell in this task description, it will only work with file logger though. 

Given people can configure their syslog to send logs to whatever storage, we can't support it here.

Reply:
I wrote this task with mind of only support log files. So I don't know how can incorporate the information with another log backend in this scenario. Maybe just add some useful comment to description or show information about another backends if there is no information about request id in logs at all.

What do you think?

Reply:
Add a guard clause for `Foreman::Logging.config[:type] == "file"` - if it is false, print what logging type is used and send the user to search for the same request id there